ngboost/distns/poisson.py

```diff
@@ -0,0 +1,67 @@
+from ngboost.distns import RegressionDistn
+from ngboost.scores import LogScore
+import scipy as sp
+import numpy as np
+from scipy.stats import poisson as dist
+from scipy.special import factorial
+from scipy.optimize import minimize
+
+### Helpers ####
+def negative_log_likelihood(params, data):
+    return -dist.logpmf(np.array(data), params[0]).sum()
```

Comment:
I see this is only used in the commented-out code.

If ultimately unused, delete.  If ultimately used, probably deserves a long-term home in `ngboost.scores` as this is not a function idiosyncratic to Poisson.